createTrie.js

```diff
@@ -99,12 +99,15 @@ function createTrie() {
 
   // private methods
   function getNodePath(query) {
-    let nodePath = [rootNode];
+    let nodePath = [];
```

Comment:
I'm not sure why I ever included `rootNode` in the node path … AFAICT it doesn't prevent any bugs or help any existing functionality.